CONTRIBUTING.md

```diff
@@ -194,11 +161,48 @@ you can change the base branch using `git rebase --onto`, like this:
   $ git rebase --onto <NEW-BASE-BRANCH> <OLD-BASE-BRANCH> <YOUR-BRANCH>
 ```
 
-For example, backporting `my_branch` from `master` to `3.x`:
+For example, backporting `my_branch` from `master` to `4.x`:
+
+```bash
+  $ git rebase --onto 4.x master my_branch
+```
+
+### Making contributions that depend on pull requests in spyder-kernels
+
+If you need to open a pull request here that has a companion pull request in the
+[spyder-kernels](https://github.com/spyder-ide/spyder-kernels) repo,
+first you need to clone the
+[git-subrepo](https://github.com/ingydotnet/git-subrepo) project
+and follow the instructions to install it (on Windows you need to use Git Bash
+in order to run the git-subrepo commands described below).
+
+Then, you need to reclone the spyder-kernels subrepo we keep here to
+point it to the clone and branch where your spyder-kernels pull request
+lives. For instance, assuming your Github user is `my-user` and your branch is
+called `my-branch`, you need to run the following command to do that:
 
 ```bash
-  $ git rebase --onto 3.x master my_branch
+git subrepo clone https://github.com/my-user/spyder-kernels.git external-deps/spyder-kernels -b my-branch -f
+```
+
+To maintain in sync the subrepo when you push more changes to `my-branch`,
+you need to run this command:
+
 ```
+git subrepo pull external-deps/spyder-kernels
+```
+
+Finally, after your pull request is merged in spyder-kernels, you need to
+reclone again its subrepo to make it point to our main repo and not to
+your clone. For that you need to use the following command:
+
+```bash
+git subrepo clone https://github.com/spyder-ide/spyder-kernels.git external-deps/spyder-kernels -b <branch> -f
+```
+
+where `<branch>` needs to be `1.x` if your Spyder pull request was done
+against our `4.x` branch, and `master` if you did it against our `master`
+branch here.
```

Comment:
Even if they are merged at the same time, the spyder-kernels subrepo is now pointing to your spyder-kernels branch, not to the main repo branch. So trying to update that subrepo afterwards would fail because no new content is going to be added to your branch.